guilford_court_commitment.yaml

```diff
@@ -0,0 +1,408 @@
+        type: "ol.effectivedate"
+        column: "EARLIEST SENTENCE EFFECTIVE DT"
+        transforms:
+        - !<transforms.DateTimeTransform>
+          pattern: ["yyyy-MM-dd HH:mm:ss.S"]
+          timezone: "America/New_York"
+    name: "sentenceterms"
+
+  sentencecomponents:
+    fqn: "ol.sentencecomponent"
+    entitySetName: "GuilfordSentenceComponents_DPS"
+    propertyDefinitions:
+      ol.id: # concat offender "nc doc id number", "commitment prefix" 
+        type: "ol.id"
+        transforms:
+        - !<transforms.ConcatTransform>
+          columns: ["OFFENDER NC DOC ID NUMBER", "COMMITMENT PREFIX"]
+      ol.status: # inmate commitment status flag
+        type: "ol.status"
+        transforms:
+        - !<transforms.BooleanRegexTransform>
+          column: "INMATE COMMITMENT STATUS FLAG"
+          pattern: "\\?+"
+          transformsIfFalse:
+          - !<transforms.ColumnTransform>
+            column: "INMATE COMMITMENT STATUS FLAG"
+          transformsIfTrue:
+          - !<transforms.ValueTransform>
+            value: ""
+      general.datetime: # commitment status date
+        type: "general.datetime"
+        column: "COMMITMENT STATUS DATE"
+        transforms:
+        - !<transforms.DateTimeTransform>
+          pattern: ["yyyy-MM-dd HH:mm:ss.S"]
+          timezone: "America/New_York"
+      ol.commitment: # commitment prefix
+        type: "ol.commitment"
+        column: "COMMITMENT PREFIX"
+    name: "sentencecomponents"
+
+  jailstays:
+    fqn: "ol.jailstay"
+    entitySetName: "GuilfordJailStays_DPS"
+    propertyDefinitions:
+      ol.id: # concat offender "nc doc id number", "offender admission/intake date"
+        type: "ol.id"
+        transforms:
+        - !<transforms.ConcatTransform>
+          columns: ["OFFENDER NC DOC ID NUMBER", "OFFENDER ADMISSION/INTAKE DATE"]
+      incident.startdatetime: # offender admission/intake date
+        type: "incident.startdatetime"
+        column: "OFFENDER ADMISSION/INTAKE DATE"
+        transforms:
+        - !<transforms.DateTimeTransform>
+          pattern: ["yyyy-MM-dd HH:mm:ss.S"]
+          timezone: "America/New_York"
+    name: "jailstays"
+
+  jailtimemodifications:
+    fqn: "ol.jailtimemodifications"
+    entitySetName: "GuilfordJailTimeCredit"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "TOTAL JAIL CREDITS (IN DAYS)"
+      ol.value: # total jail credit in days
+        type: "ol.value"
+        column: "TOTAL JAIL CREDITS (IN DAYS)"
+      ol.units: # value transform and write in days
+        type: "ol.units"
+        transforms:
+        - !<transforms.ValueTransform>
+          value: "Days"
+    name: "jailtimemodifications"
+
+  charges:
+    fqn: "ol.charge"
+    entitySetName: "GuilfordCharges"
+    propertyDefinitions:
+      ol.id: # most serious offense code
+        type: "ol.id"
+        transforms:
+        - !<transforms.BooleanRegexTransform>
```

Comment:
This could be shorter if you judt replace `\\?+` with ``.

Reply:
yep, not sure what I was thinking except for confusion. replacing all instances of this pattern with:
` column: "MOST SERIOUS OFFENSE CODE"
        transforms:
        - !<transforms.ReplaceRegexTransform>
          target: "\\?+"
          goal: ""`